frontend/packages/core/src/stories/linearTimeline.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { Meta } from "@storybook/react";
+
+import type { LinearTimelineData } from "../Charts/linearTimeline";
+import LinearTimeline from "../Charts/linearTimeline";
+
+export default {
+  title: "Core/LinearTimeline",
+  component: LinearTimeline,
+} as Meta;
+
+const ChartContainer = styled("div")({
+  width: 400,
+  height: 200,
+});
+
+export const Primary = () => {
+  const mockData: LinearTimelineData = {
+    deploys: { points: [{ timestamp: 1588884888 }], shape: "cross", color: "purple" },
+    "k8s events": { points: [{ timestamp: 1588985888 }], shape: "triangle", color: "blue" },
+    explosions: {
+      points: [
+        { timestamp: 1588788888 },
```

Comment:
is it possible to annotate individual events with tooltips? i looked at the recharts docs briefly for `Scatter` but it wasn't clear to me what the possibilities were for the `data` prop. also was reading https://stackoverflow.com/a/66084216

Reply:
yes it is. The payload object contains all the data for the point so with custom tooltip renders we can put whatever we want. The thing that seemed difficult was that there was no default styling, so if you don't pass any styling info the text is just black with no background or box. We can fix that by passing styling info.